mobile-credential-online-presentation/README.md

```diff
@@ -0,0 +1,104 @@
+> [!NOTE]
+> This is a tech preview. If you are interested in in trialling this capability, please reach out via [https://mattr.global/contact-us](https://mattr.global/contact-us).
+
+# Mobile Credential Online Presentation Sample App
+
+This sample app demonstrates the usage of the Web Verifier SDK to present mobile credentials online on the same device and across devices.
+
+## Prerequisites
+
+### MATTR verifier tenant configuration
+
```

Comment:
We can probably link to the tech-preview API Ref instead of detailing all of this here? 

https://online-presentations-tech-preview.redoc.ly/tag/Mobile-Credentials-Verification#operation/putVerifierConfiguration

Then just say something like

You must create a Verifier configuration on your MATTR VII verifier tenant. Refer to the API reference for a complete description of this configuration and different settings you can control.

Also might be worth calling out that they need to sign up for a trial tenant if they don't have access to one?

Reply:
Fixed in https://github.com/mattrglobal/sample-apps/pull/167/commits/776c8c6240f228f1fbf937a92be937ecf460594f.

I did not specifically call out signing up for a trial tenant, because the banner on top of the README calls out getting in touch anyways. Trial tenants would not have online presentation enabled by default.